src/main/resources/application-default.yml

```diff
@@ -0,0 +1,59 @@
+#========================================
+# General settings
+#========================================
+integration:
+  party:
+    connect-timeout: 5
+    read-timeout: 30
+logging:
+  level:
+    root: INFO
+openapi:
+  name: ${spring.application.name}
+  title: ${spring.application.name}
+  version: '@project.version@'
+spring:
+  application:
+    name: api-alkt
+  datasource:
+    configuration:
+      maximum-pool-size: 10
+      connection-timeout: 5000
+      read-only: true
+    url: jdbc:sqlserver://localhost:1433;databaseName=AlkTSundsvall;trustServerCertificate=true
+    username: SA
+    password: P@ssword
+    driver-class-name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+  jpa:
+    hibernate:
+      naming:
+        physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
+        implicit-strategy: org.hibernate.boot.model.naming.ImplicitNamingStrategyLegacyJpaImpl
+    properties:
+      hibernate:
+        connection:
+          readOnly: true
+        jdbc:
+          time_zone: Europe/Stockholm
+        hbm2ddl:
+          schema-generation:
+            script:
+              append: false
+        default_schema: dbo
+      jakarta:
+        persistence:
+          schema-generation:
+
```

Comment:
Weird void?

Reply:
fudge, that file wasn't meant to even be committed

Reply:
Add to .gitignore